conjurapi/config_test.go

```diff
@@ -26,75 +27,70 @@ func TempFileForTesting(prefix string, fileContents string) (string, error) {
 }
 
 func TestConfig_IsValid(t *testing.T) {
-	Convey("Return without error for valid configuration", t, func() {
+	t.Run("Return without error for valid configuration", func(t *testing.T) {
 		config := Config{
 			Account:      "account",
 			ApplianceURL: "appliance-url",
 		}
 
 		err := config.validate()
-
-		So(err, ShouldBeNil)
+		assert.NoError(t, err)
 	})
 
-	Convey("Return error for invalid configuration", t, func() {
+	t.Run("Return error for invalid configuration", func(t *testing.T) {
 		config := Config{
 			Account: "account",
 		}
 
 		err := config.validate()
-		So(err, ShouldNotBeNil)
+		assert.Error(t, err)
 
 		errString := err.Error()
-
-		So(errString, ShouldContainSubstring, "Must specify an ApplianceURL")
+		assert.Contains(t, errString, "Must specify an ApplianceURL")
 	})
 }
 
 func TestConfig_IsHttps(t *testing.T) {
-	Convey("Return true for configuration with SSLCert", t, func() {
+	t.Run("Return true for configuration with SSLCert", func(t *testing.T) {
 		config := Config{
 			SSLCert: "cert",
 		}
 
-		err := config.IsHttps()
-
-		So(err, ShouldBeTrue)
+		isHttps := config.IsHttps()
```

Comment:
Seems like several test cases can be combined into a table, e.g. some test cases call `config.validate`, others call `config.IsHttps()`, some with good config, some with invalid config.